spec/support/shared_examples_for_providers.rb

```diff
@@ -46,7 +49,10 @@
 
   context 'uncaught in a model' do
     it 'reports the error to Rollbar' do
-      Rollbar.should_receive(:log).with('error', instance_of(RuntimeError)).and_return({})
+      Rollbar.should_receive(:log) do |level, args|
+        expect(level).to eq('error')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.